Extension/src/logger.ts

```diff
@@ -98,6 +99,15 @@ export function getDiagnosticsChannel(): vscode.OutputChannel {
     return diagnosticsChannel;
 }
 
+export function getCrashCallStacksChannel(): vscode.OutputChannel {
+    if (!crashCallStacksChannel) {
+        crashCallStacksChannel = vscode.window.createOutputChannel(localize("c.cpp.crash.call.stacks.title", "C/C++ Crash Call Stacks"));
+        crashCallStacksChannel.appendLine(localize("c.cpp.crash.call.stacks.description",
+            "The information below could be helpful to provide in a C/C++ extension bug report:"));
```

Comment:
Would mentioning `on GitHub` (and/or a `https://github.com/microsoft/vscode-cpptools/issues` URL) be better?

Reply:
Yes, a URL is helpful.  I would also change `could` to `would` since we'd like to link crash reports to actual code as often as we can.

Reply:
@bobbrow Sure, I can add the URL, which I think would allow users to click/navigate to it to. Using text "on GitHub" seems sort of superfluous given that the URL is a github.com one.